components/brave_wallet/browser/rlp_encode.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_RLP_ENCODE_H_
```

Comment:
invalid ifndef name

Reply:
Fixed by interactive rebase in the next push